python/mach_bootstrap.py

```diff
@@ -143,6 +146,20 @@ def _activate_virtualenv(topdir):
         os.path.join("tests", "wpt", "harness", "requirements_servo.txt"),
     ]
 
+    if need_pip_upgrade:
+        # Upgrade pip when virtualenv is created to fix the issue
+        # https://github.com/servo/servo/issues/11074
+        pip = _get_exec_path(PIP_NAMES, is_valid_path=check_exec_path)
+        if not pip:
+            sys.exit("Python pip is either not installed or not found in virtualenv.")
```

Comment:
I don't really know. @broadcrawford added such check on Sep 18, 2015 for all package installations (you can check them in the code below). I just copy-pasted them.